gprof2dot.py

```diff
@@ -56,6 +56,25 @@ def fail(a, b):
     assert False
 
 
+def round_difference(difference, tolerance):
+    n = int(str(tolerance).split('-')[1]) if '-' in str(tolerance) else len(str(tolerance).split('.')[1])
```

Comment:
Could you add a brief comment explaining the intent of  this function?  It's not clear.
I'd prefer you used `math.floor` or `math.ceil` instead of integer <-> string conversions.